bolt-design-system/bolt
# Fix: vertical alignment of text and icon in card button

## Jira

Part of this issue but it does not address the `verylongunbrokenword`: http://vjira2:8080/browse/WWWD-2680

## Summary

Fixes an issue when 2 buttons are used in a card, the lesser text button would align incorrectly. This makes text and icon align center vertically.

## Details

`align-items: center` is added to the bolt-button CSS, which fixes the vertical alignment. I also updated the demo pages to reflect the latest changes.

Screenshot of before and after:

![screen shot 2018-10-15 at 4 50 40 pm](https://user-images.githubusercontent.com/3027663/46979626-12968800-d0a0-11e8-99e3-2cedc2471603.png)

## How to test

Pull down the branch and run it locally, check out the card with two buttons demo, adjust the width to get the longer text to wrap.
